3_visualize/src/drought_learner_viz_components.R

```diff
@@ -156,11 +156,11 @@ frame_a <- function(blank_plot, streamflow_df, droughts_df,
   main <- blank_plot +
     # Fixed threshold drought durations
     annotate("rect", # fixed threshold
-             xmin = droughts_df$start[droughts_df$method == "fixed"],
-             xmax = droughts_df$end[droughts_df$method == "fixed"],
+             xmin = droughts_df$start[droughts_df$method == "fixed" & droughts_df$drought_id == 76],
+             xmax = droughts_df$end[droughts_df$method == "fixed" & droughts_df$drought_id == 76],
```

Comment:
I rectified the bottom bars problems by making sure only the first, August drought shows for several frames, then the fixed droughts in general, then the variable droughts added in, and then finally the year-extent of bottom bars are on the final frame. To do this, I created a more global bottom_bars function that makes these targets specifically, rather than adding in a separate function within the `frame_m()` function as it was before when the extent changed to the full year.

Reply:
ooh nice to have all the bottom bars defined by a single function. Good fix!